src/components/Atoms/CheckBox/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import Check from "../../../assets/svg/check.svg";
+
+const CheckBox = ({ onClick }) => {
+  return (
+    <StyledCheckBox onClick={onClick}>
+      <HoverCheck src={Check}></HoverCheck>
```

Comment:
```suggestion
      <HoverCheck src={Check}/>
```
閉じタグ省略できるところはどんどんしていく感じで問題ないと思います